src/ui/qgselevationshadingrenderersettingswidget.ui

```diff
@@ -0,0 +1,216 @@
+           <property name="toolTip">
+            <string>Used multidirectionnal lights instead of the specified azimuth for hillshading.</string>
+           </property>
+           <property name="text">
+            <string/>
+           </property>
+           <property name="tristate">
+            <bool>false</bool>
+           </property>
+          </widget>
+         </item>
+         <item row="2" column="0" colspan="2">
+          <widget class="QgsDirectionalLightWidget" name="mDirectionalLightWidget" native="true"/>
+         </item>
+        </layout>
+       </widget>
+      </item>
+      <item row="2" column="0" colspan="2">
+       <widget class="QgsCollapsibleGroupBox" name="mEdlGroupBox">
+        <property name="title">
+         <string>Eye-Dome Lighting</string>
+        </property>
+        <property name="checkable">
+         <bool>true</bool>
+        </property>
+        <layout class="QGridLayout" name="gridLayout_5">
+         <item row="0" column="1">
+          <widget class="QgsDoubleSpinBox" name="mEdlStrengthSpinBox">
+           <property name="toolTip">
+            <string>Adjusts how strong the added shading will be. Greater values mean stronger effect.</string>
+           </property>
+           <property name="decimals">
+            <number>0</number>
+           </property>
+           <property name="maximum">
+            <double>999999999999.000000000000000</double>
+           </property>
+           <property name="singleStep">
+            <double>100.000000000000000</double>
+           </property>
+           <property name="value">
+            <double>1000.000000000000000</double>
+           </property>
+          </widget>
+         </item>
+         <item row="1" column="1">
+          <widget class="QgsDoubleSpinBox" name="mEdlDistanceSpinBox">
+           <property name="toolTip">
+            <string>Distance away from the original pixel to sample neighbors.</string>
+           </property>
+          </widget>
+         </item>
+         <item row="1" column="0">
+          <widget class="QLabel" name="label_2">
+           <property name="text">
+            <string>Distance</string>
+           </property>
+          </widget>
+         </item>
+         <item row="0" column="0">
+          <widget class="QLabel" name="label">
+           <property name="text">
+            <string>Strength</string>
+           </property>
+          </widget>
+         </item>
+         <item row="1" column="2">
+          <widget class="QgsUnitSelectionWidget" name="mEdlDistanceUnit" native="true"/>
+         </item>
+        </layout>
+       </widget>
+      </item>
+      <item row="0" column="0" colspan="2">
+       <widget class="QLabel" name="label_4">
+        <property name="text">
+         <string>Apply shading on the canvas following elevation of Point Cloud layers, Mesh layers and Raster layers if they represents elevation surface.</string>
```

Comment:
s/represents/represent/

maybe also use lowercase for point cloud, mesh, raster layers...